### DISCUSSION TITLE
Improve default legibility of disabled field interfaces

### DISCUSSION THREAD

**Discussion by colemahatmccreary** [directus/directus#15286] [Feature Requests]
I understand this is fully customizable with an on prem instance, but when using Directus Cloud the grey on grey is _pretty_ bad. 

Can we PLEASE get a simple hex update to increase contrast between text and background??

**Comment by benhaynes** [comment]
You should be able to use Settings > CSS Overrides to update the necessary CSS variable(s) for this.

**Comment by colemahatmccreary** [comment]
Ah, this is true @benhaynes. 

But frankly the global defaults should be updated platform-wide, as the current contrast is actually an accessibility issue (see  https://webaim.org/resources/contrastchecker/)

Plus the greys used appear to be different depending on the interface/field type (text vs m2o, etc), which suggests that there are multiple sets of variables that need to change. Would rather not start pulling that thread if i can avoid it as I haven’t delved into where different directus styles/variables are used.

**Reply by benhaynes** [reply]
I agree! We are in process of adding and standardizing the variables, naming conventions, and a theming system... so I can't wait to get this one pushed through. @rijkvanzanten I know we don't have time right now, but how concerned are you about these PRs getting stale?

https://github.com/directus/directus/pull/13247
https://github.com/directus/directus/pull/13248

**Reply by rijkvanzanten** [reply]
> but how concerned are you about these PRs getting stale?

Ehh usual amounts of concerned? It's the same as with other big PRs that touch a lot of files, they get stale fast.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
